oor_on_edge/data_delivery_pipeline/components/data_delivery.py

```diff
@@ -137,8 +133,11 @@ def _deliver_data_batch(
                 detection_metadata_full_path = detections_path / pathlib.Path(
                     f"{image_full_path.stem}.txt"
                 )
-
-                if os.path.isfile(image_full_path) and os.path.isfile(
+                gps_lat = float(row[12])
```

Comment:
We might want to make a backlog ticket to switch to a more robust CSV reading method for all on_edge code such as pandas. Now we have a bunch of these magic column indices that will give us a headache if the metadata format changes at some point.